vm/devices/storage/nvme_spec/src/nvm.rs

```diff
@@ -16,7 +16,7 @@ use zerocopy::LE;
 use zerocopy::U16;
 
 #[repr(C)]
-#[derive(Debug, AsBytes, FromBytes, FromZeroes, Inspect)]
+#[derive(Debug, AsBytes, FromBytes, FromZeroes, Inspect, Clone)]
```

Comment:
can you share what fails to compile (if anything) if you remove Clone?

Reply:
Fails in namespace.rs line 85 - when we restore namespace object if Identify structure was provided from saved state

Reply:
lots of code has changed in 2 weeks - bumping to check if this is still necessary (and if so - what code requires the Clone bound)

Reply:
@daprilik it is used in NvmeDriver::restore even though there is a new direction to re-query Identify Namespace instead, I think we should keep that code for future improvements. If you see a better way to implement that without clone() please let me know.